bin/photos.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+'use strict'
+
+var easyimage = require('easyimage')
+var fs = require('fs')
+
+fs.readdir('./images/gallery', function (err, items) {
```

Comment:
So the current flow is drop the big images into the gallery folder, run the photos script to generate the thumbnails and then remove the large images again? 

It seems simpler to just add the image urls into the json file. 

I can imagine a situation where someone forgets to delete the original images and we end up with lots of large images in the repo. 

If this will be fixed in a subsequent PR I don't mind merging but only as a temporary measure.